staging/src/k8s.io/apiserver/pkg/apis/config/types.go

```diff
@@ -33,7 +33,20 @@ type EncryptionConfiguration struct {
 
 // ResourceConfiguration stores per resource configuration.
 type ResourceConfiguration struct {
-	// resources is a list of kubernetes resources which have to be encrypted.
+	// resources is a list of kubernetes resources which have to be encrypted. The resource names are derived from `resource` or `resource.group` of the gvr.
+	// eg: pandas.awesome.bears.example is a custom resource with 'group': awesome.bears.example, 'resource': pandas)
+	// example:
+	// apiVersion: apiserver.config.k8s.io/v1
+	// kind: EncryptionConfiguration
+	// resources:
+	// - resources:
```

Comment:
maybe limit the example to the values in the `resources` field, rather than an entire config file?

Reply:
Can we include the entire config example for the actual `EncryptionConfiguration` struct and not a field?

Reply:
done. PTAL